core/src/main/kotlin/org/evomaster/core/problem/rest/service/BlackBoxRestFitness.kt

```diff
@@ -33,8 +35,13 @@ class BlackBoxRestFitness : RestFitness() {
             rc.resetSUT()
         }
 
-        cookies.putAll(getCookies(individual))
-        tokens.putAll(getTokens(individual))
+        //val cookies = AuthUtils.getCookies(client, getBaseUrl(), individual)
+       // val tokens = AuthUtils.getTokens(client, getBaseUrl(), individual)
+
+        //cookies.putAll(getCookies(individual))
+        //tokens.putAll(getTokens(individual))
```

Comment:
remove commented out code

Reply:
Hi Andrea,

Thanks for your comment. I have removed those commented codes.